aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -74,16 +84,17 @@
     @Before
     public void setup() {
         this.context = getApplicationContext();
-        modelProvider = spy(SimpleModelProvider.builder()
-            .version(RandomString.string())
-            .addModel(Person.class)
-            .build());
+        modelProvider = spy(AmplifyCliGeneratedModelProvider.singletonInstance());
+        subscriptionCancelledCounter = new AtomicInteger(0);
+        subscriptionStartedCounter = new AtomicInteger(0);
```

Comment:
0 is already the default for [`AtomicInteger()`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/AtomicInteger.html#AtomicInteger--)